service/test/controllers/courseenrollment/CourseEnrollmentControllerTest.java

```diff
@@ -4,90 +4,211 @@
 import static org.junit.Assert.assertEquals;
 import static play.test.Helpers.route;
 
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
 import com.fasterxml.jackson.databind.JsonNode;
+import controllers.BaseController;
 import controllers.BaseControllerTest;
+
+import java.io.File;
+import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
-import org.junit.FixMethodOrder;
-import org.junit.Test;
-import org.junit.Ignore;
+
+import controllers.DummyActor;
+import modules.OnRequestHandler;
+import modules.StartModule;
+import org.junit.*;
 import org.junit.runner.RunWith;
 import org.junit.runners.MethodSorters;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
 import org.powermock.core.classloader.annotations.PowerMockIgnore;
 import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.core.classloader.annotations.SuppressStaticInitializationFor;
 import org.powermock.modules.junit4.PowerMockRunner;
 import org.sunbird.common.models.util.JsonKey;
+import play.Application;
+import play.Mode;
+import play.inject.guice.GuiceApplicationBuilder;
 import play.libs.Json;
 import play.mvc.Http;
 import play.mvc.Http.RequestBuilder;
 import play.mvc.Result;
+import play.test.Helpers;
 import util.RequestInterceptor;
 
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 @RunWith(PowerMockRunner.class)
-@PrepareForTest(RequestInterceptor.class)
+@PrepareForTest({OnRequestHandler.class})
+@SuppressStaticInitializationFor({"util.AuthenticationHelper", "util.Global"})
 @PowerMockIgnore("javax.management.*")
-@Ignore
-public class CourseEnrollmentControllerTest extends BaseControllerTest {
-  public static final String UNENROLL_URI = "/v1/course/unenroll";
-  public static final String ENROLL_URI = "/v1/course/enrol";
+public class CourseEnrollmentControllerTest {
 
   public static String COURSE_ID = "courseId";
   public static String BATCH_ID = "batchId";
   public static String USER_ID = "userId";
+  private static final String ENROLL_BATCH_URL = "/v1/course/enroll";
+  private static final String UENROLL_BATCH_URL = "/v1/course/unenroll";
+  private static final String GET_ENROLLED_COURSES_URL = "/v1/user/courses/list/"+USER_ID;
+  private static final String GET_ENROLLED_COURSE_URL = "/v1/user/courses/read";
+  public static Application application;
+  public static ActorSystem system;
+  public static final Props props = Props.create(DummyActor.class);
```

Comment:
can we avoid making these field static? it should work even if these are instance fields. please verify and confirm